### PR TITLE
Cleanup codyco-superbuild references

### PR DESCRIPTION
**Pull request by traversaro**

### CODE REVIEW

File: torque-controllers/momentum-based-standup/README.md [hunk 1]
```diff
@@ -10,7 +10,7 @@ In order to run the simulation of iCub standing up from a chair, make sure you h
 
  - Both iCub and the chair models are stored into [icub-gazebo-wholebody](https://github.com/robotology-playground/icub-gazebo-wholebody) repository.
 
- It is suggested to install the above mentioned repository using [codyco-superbuild](https://github.com/robotology/codyco-superbuild) (enable `CODYCO_USES_GAZEBO`, `CODYCO_USES_MATLAB`, options).
+ It is suggested to install the above mentioned repository using [robotology-superbuild](https://github.com/robotology/robotology-superbuild) (enable `ROBOTOLOGY_ENABLE_DYNAMICS`, `CODYCO_USES_GAZEBO` and `CODYCO_USES_MATLAB` options).
```
Comment (gabrielenava) [inline torque-controllers/momentum-based-standup/README.md]: I guess also `CODYCO_USES_GAZEBO` and `CODYCO_USES_MATLAB` must be modified.

**Review by gabrielenava — CHANGES_REQUESTED** [review]

**Comment by gabrielenava** [comment]
Sorry, I reviewed this days ago but didn't realize my review was not submitted.

File: torque-controllers/momentum-based-standup/README.md (hunk 1 above)
Comment (traversaro) [reply torque-controllers/momentum-based-standup/README.md]: Done in https://github.com/robotology/whole-body-controllers/pull/32/commits/8e5e944d11c2b04237b32b779b5f2d3aa12ea1d8 .

**Review by gabrielenava — APPROVED** [review]

**Comment by gabrielenava** [comment]
Merging.

**Pull request merged**